### PR TITLE
Feature/cli purge

### PR DESCRIPTION
**Pull request by zelima**
`data purge [-f]` command for deleting data packages

- Prompts user before deleting 
- Does not prompt if -f of --force provided 
- help 
- tests

### CODE REVIEW

File: lib/purge.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    prompt.get(schema, async (err, result) => {
+      if (err) {
+        return console.error(err.message+'\n');
+      }
+      const token = await getToken(config)
+      await requestPurge(config, dpjson.name, token)
+      console.log(`Successfully deleted ${dpjson.name}`)
+    })
+  } else {
+    const token = await getToken(config)
```
Comment (zelima) [inline lib/purge.js]: I repeat myself here and doing same exact thing twice (lin37,38,39) but could not find any better way to await prompt. Feel free to add commit fixing this

File: lib/purge.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  const token = await getToken(config)
+  const resp = await requestPurge(config, dpjson.name, token)
+  if (resp.status === 'OK') {
+    console.log(`Successfully deleted ${dpjson.name}`)
+  } else {
+    console.log(`Failed to deleted ${dpjson.name}`)
+  }
+}
+
+const getToken = async(config) => {
```
Comment (anuveyatsu) [inline lib/purge.js]: It seems like we have these functions in our branch for `push` command, so in the future, we need to have them somewhere in the `utils` and import them as needed to be as DRY as possible.

Probably, we could start doing it from here?

File: lib/purge.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        console.error(err.response.data.message)
+        process.exit(1)
+      }
+      console.error(err.message)
+      process.exit(1)
+    })
+  return res.data
+}
+
+const checkDpIsThere = () => {
```
Comment (anuveyatsu) [inline lib/purge.js:96]: This function as well.

**Review by anuveyatsu — COMMENTED** [review]
I have pushed a small commit on making code less repetitive. However, I believe that entire thing about checking `-f` flag and prompting a user to type package name should go into `bin/data-purge.js`. Would you agree?

**Comment by zelima** [comment]
@anuveyatsu yes. Only reason I'm doing this inside lib is that I'm displaying data package name when prompting the warning. We have three options:
- take out logic into bin and get dp name there
- leave as is
- not display dp name in prompt at all

**Comment by anuveyatsu** [comment]
@zelima well, I'm not sure how significant is this so we could leave it as is for now.

**Comment by zelima** [comment]
@anuveyatsu ok merging for now, we can come back to this later

**Pull request merged**